developers_chamber/qa/checks.py

```diff
@@ -26,7 +26,7 @@ class MigrationFilenamesQACheck(QACheck):
     name = 'Check migration filenames'
 
     def _is_migration_file_with_wrong_name(self, path):
-        return self._is_migration_file(path) and not re.search(r'/([0-9]{4}_migration|__init__)\.py$', path)
+        return self._is_migration_file(path) and not re.search(r'/([0-9]{4}_migration|__init__)\.py$', str(path))
```

Comment:
What type is `path`?

Reply:
I am not sure, maybe its instance of `Path`, nevertherless, `search` was complaining, that this argument must be string or bytes (it probably does not call it's __str__ method implicitly).